frontend/src/pages/Services/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Typography, Stack } from "@mui/material";
+import LocalShippingOutlinedIcon from "@mui/icons-material/LocalShippingOutlined";
+import ShoppingBagOutlinedIcon from "@mui/icons-material/ShoppingBagOutlined";
+import StarBorderOutlinedIcon from "@mui/icons-material/StarBorderOutlined";
+import { Box1Styled, BoxIconStyled, BoxStyled } from "./styles";
+
+const Services = () => {
+  return (
+    <div>
+      <BoxStyled>
+        <Typography variant="h3">Why Shop With Us</Typography>
+        <Stack direction="row" justifyContent="center" spacing={3}>
+          <Box1Styled>
+            <Typography>
+              <BoxIconStyled>
+                <LocalShippingOutlinedIcon
+                  sx={{ width: "55px", height: "auto" }}
+                />
+                <Typography variant="h5">Fast Delivery </Typography>
+                <Typography
+                  variant="body2"
+                  sx={{
+                    margin: "auto",
+                    width: "300px",
+                    textAlign: "center",
+                    wordWrap: "break-word",
+                  }}
+                >
```

Comment:
avoid inline styles, instead create `styles.ts` file and add related styles inside the file as objects, which you can use here in this component as `styles.typograpgy`.